### PR TITLE
More idiomatic java

### PR DESCRIPTION
**Pull request by DylanMeeus**
I changed some function calls to make it more idiomatic to Java.

There are some more things in the code that are not ideomatic (the way you should do it in Java) but I felt it might be inapropriate to just start changing things around. Do you mind if I make some changes?

### CODE REVIEW

File: src/main/java/ovh/not/javamusicbot/ShardManager.java [hunk 1]
```diff
@@ -24,11 +24,10 @@
     ShardManager(int shardCount, int minShard, int maxShard) {
         shards = new Shard[(maxShard - minShard) + 1];
         int index = 0;
-        for (int shardId = minShard; shardId < maxShard + 1;) {
+        for (int shardId = minShard; shardId < maxShard + 1;shardId++) {
```
Comment (bramhaag) [inline src/main/java/ovh/not/javamusicbot/ShardManager.java]: Missing space

Comment (DylanMeeus) [reply src/main/java/ovh/not/javamusicbot/ShardManager.java]: I've added the space. 
And actually just combined the relevant variables inside the loop declaration itself.

**Comment by ducc** [comment]
¯\_(ツ)_/¯

**Comment by bramhaag** [comment]
Looks like you've lost an arm @ducc :/

**Comment by ducc** [comment]
Change whatever you like, we will happily take all the help we can get :)

**Comment by DylanMeeus** [comment]
@ducc thank you :-) cool project btw ;-)

**Pull request merged**